ql/src/test/results/clientpositive/llap/dynpart_sort_optimization.q.out

```diff
@@ -161,20 +168,28 @@ STAGE PLANS:
                             Map-reduce partition columns: _col0 (type: string), _col1 (type: tinyint)
                             Statistics: Num rows: 1 Data size: 24 Basic stats: COMPLETE Column stats: NONE
                             value expressions: _col2 (type: smallint), _col3 (type: smallint), _col4 (type: bigint), _col5 (type: bigint), _col6 (type: binary), _col7 (type: int), _col8 (type: int), _col9 (type: bigint), _col10 (type: binary), _col11 (type: bigint), _col12 (type: bigint), _col13 (type: bigint), _col14 (type: binary), _col15 (type: float), _col16 (type: float), _col17 (type: bigint), _col18 (type: binary)
-                      Reduce Output Operator
-                        key expressions: _col4 (type: tinyint)
-                        null sort order: a
-                        sort order: +
-                        Map-reduce partition columns: _col4 (type: tinyint)
-                        Statistics: Num rows: 1 Data size: 24 Basic stats: COMPLETE Column stats: NONE
-                        value expressions: _col0 (type: smallint), _col1 (type: int), _col2 (type: bigint), _col3 (type: float)
             Execution mode: llap
             LLAP IO: all inputs
         Reducer 2 
+            Execution mode: llap
```

Comment:
could you please clarify why the patch has moved this operator from Reducer3 to Reducer2? basically, I "like" that we eliminated a reducer stage, however, I don't have an idea whether this is correct not, or what's the root cause behind it? 

as a reference, the history of dynamic partitioning sort optimization (what I can recall) is: HIVE-6455, HIVE-20703, HIVE-23071

Reply:
Actually we did not eliminate anything. If you look closely, what happened is Reducer 2 and Reducer 3 exchanged names. We still have the same number of mappers/reducers and the same operators overall. I am not sure why the names changed however they are arbitrary anyway. So I am not concerned with that.

Reply:
okay, thanks!